brd-ios/breadwallet/src/Redesign/Scenes/Swap/Workers/ExchangeHistoryWorker.swift

```diff
@@ -11,38 +11,54 @@
 import Foundation
 
 struct ExchangesHistoryResponseData: ModelResponse {
-    var exchanges: [ExchangeHistoryResponseData]
+    var exchanges: [ExchangeResponseData]
 }
 
-struct ExchangeHistoryResponseData: ModelResponse {
-    enum Status: String, Codable {
-        case pending = "PENDING"
-        case complete = "COMPLETE"
-        case failed = "FAILED"
+struct ExchangeResponseData: ModelResponse {
+    struct SourceDestination: ModelResponse {
```

Comment:
could the existing SwapResponseData / Swap models be used?
have a feeling they should be the same objects